openapi.yaml

```diff
@@ -1525,6 +1525,21 @@ paths:
                         - returns
                       allOf:
                         - $ref: '#/components/schemas/process'
+                  namespaces:
+                    type: array
+                    description: |-
+                      A list of namespaces the user has access too.
+
+                      The following namespaces are usually listed:
+                      * `backend`
+                      * `user`: only if authenticated
+                      * `@` + other user id: A list of namespaces of *other* users
+                        which share at least one process accessible by the user.
+                        The namespace with the authenticated user id MUST NOT be listed.
+
+                      For non-authenticated users public processes are taken into account.
```

Comment:
How are "public" processes defined at this point? Is that just pre-defined processes?

Reply:
This mixes the current state and sharing a bit, so if you have a public: true/false flag for sharing, all public processes would be returned.

Reply:
ok so this to be compatible with the still to be discussed sharing aspect?